crates/sui-indexer-alt-graphql/src/scope.rs

```diff
@@ -161,3 +166,11 @@ impl Scope {
         Resolver::new_with_limits(self.package_store.clone(), self.resolver_limits.clone())
     }
 }
+
+impl Debug for Scope {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        write!(f, "checkpoint_viewed_at={:?}", self.checkpoint_viewed_at)?;
+        write!(f, "root_version={:?}", self.root_version)?;
+        write!(f, "resolver_limits={:?}", self.resolver_limits)
```

Comment:
If you want to keep around a `Debug` impl for `Scope`, it should look like this, so it's consistent with derived `Debug` impls.

```suggestion
        f
            .debug_struct("Scope")
            .field("checkpoint_viewed_at", &self.checkpoint_viewed_at)
            .field("root_version", &self.root_version)
            .field("resolver_limits", &self.resolver_limits)
            .finish()
```